### PR TITLE
Role arn doc

### PR DESCRIPTION
**Pull request by ofer-haim**
Added instructions reg. role IAM settings

### CODE REVIEW

**Review by hadarkorny — APPROVED** [review]

**Review by RefaelAdi — APPROVED** [review]

File: input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+			"Action": "sts:AssumeRole"
+		}
+		]
+	}
+   ```
+
+10. Set the role created on first AWS Account i.e., role_on_111111 to the EC2 machine hosting Guardium
+11. Go to the EC2 machine hosting Guardium
+12. Right-click on the EC2 instance, select the Security Option, and modify the IAM role
+13. Set the role that was created above
```
Comment (AlexaNahum) [inline input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md:222]:
```suggestion
13. Set the role that was created previously.
```

File: input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+
+## Configuring input plugin on Guardium:
```
Comment (AlexaNahum) [inline input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md:225]:
```suggestion
## Configuring an input plug-in on Guardium:
```

File: input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+
+### Procedure :
+
+Update the input section to add the details from the corresponding file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
+
+The sample configuration looks like :-
```
Comment (AlexaNahum) [inline input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md:231]:
```suggestion
The following is a sample configuration:
```

File: input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
+## Configuration for role_arn parameter in the input plug-in
+
+For input plug-ins like cloudwatch_logs and sqs, the AWS credentials are required to access the AWS endpoint either cloudwatch_logs log_group or sqs queue. However, instead of using the AWS access_key and secret_key, using the role_arn setting can be preferable.
+
+### Note:
+
+These settings can be used only when the Guardium Data Protection is hosted on AWS.
```
Comment (AlexaNahum) [inline input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md:7]:
```suggestion
These settings can be used only when Guardium Data Protection is hosted on AWS.
```

File: input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+21. Right click on the EC2 instance, select the Security Option, and modify the IAM role
+22. Set the role that was created above
+
+## Configuration for IAM Role when the Guardium Data Protection and the Database to be monitored are in different AWS accounts
+
+### Prerequisites :
+
+1.  AWS Account1 where IBM Guardium is hosted on EC2 instance. For simplicity consider this as Account with Account ID 111111
+2.  AWS Account2 where RDS is present. For simplicity consider this as Account with Account ID 222222
+3.  Log group in the Account2 where RDS logs are logged. For simplicity consider test-log-group is the log group that is logging the logs in the region us-east-1
```
[the remark this answers is not in the document]
Comment (AlexaNahum) [reply input-plugin/logstash-input-cloudwatch-logs/SettingsForRoleArn.md:83]: A log group in the Account2 where RDS logs are logged. In this example, the test-log-group is in region us-east-1.

**Review by AlexaNahum — APPROVED** [review]

**Review by AlexaNahum — APPROVED** [review]

**Pull request merged**